packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -815,6 +896,7 @@ void main() {
         headerBuilder: (BuildContext context, bool isExpanded) {
           return const Text('Collapsed', key: collapsedKey);
         },
+        canTapOnHeader: true,
```

Comment:
This should probably be a separate new test since it changes the semantics?

Reply:
I removed this part, I think this is probably not necessary because the new test I added, 'ExpansionPanelList does not merge header when canTapOnHeader is false',
should cover this case already